### PR TITLE
Fix function names and documentation

### PR DESCRIPTION
**Pull request by ravanelli**
Signed-off-by: Renata Ravanelli <rravanel@redhat.com>

### CODE REVIEW

File: vars/gangplank.groovy [hunk 1]
```diff
@@ -46,19 +46,28 @@ def generateSpec(params = [:]) {
...
-    gangplankCmd = getFlags(params, gangplankCmd)
+    gangplankCmd = _getArtifacts(params, gangplankCmd)
+    gangplankCmd = _getFlags(params, gangplankCmd)
     fileName = "/tmp/jobspec-${UUID.randomUUID()}.spec"
     gangplankCmd += "--yaml-out ${fileName} "
     runGangplank(gangplankCmd)
     return fileName
 }
-
+// Available parameters:
```
Comment (jlebon) [inline vars/gangplank.groovy:56]: Minor: missing space before this line.

File: vars/gangplank.groovy [hunk 2]
```diff
@@ -8,8 +8,8 @@
 //     workDir:      string  -- cosa working directory
 
 def buildArtifact(params = [:]) {
-    gangplankCmd = getMode(params)
-    gangplankCmd = getArtifacts(params, gangplankCmd)
+    gangplankCmd = _getMode(params)
+    gangplankCmd = _getArtifacts(params, gangplankCmd)
```
[the remark this answers is not in the document]
Comment (jlebon) [reply vars/gangplank.groovy:12]: More of a convention. It's still callable in the end, so I guess like Python. :)

File: vars/gangplank.groovy [hunk 3]
```diff
@@ -46,19 +46,28 @@ def generateSpec(params = [:]) {
...
     runGangplank(gangplankCmd)
 }
 
+// Available parameters:
+//     artifacts:    list    -- list of artifacts to be built
+//     extraFlags    string  -- Extra flags to use
+//     image:        string  -- Name of the image to be used
+//     mode:         string  -- Gangplank Mode
+//     minioServeDir string  -- location to service minio from
+//     workDir:      string  -- cosa working directory
```
Comment (jlebon) [inline vars/gangplank.groovy:70]: Super minor: looks like we're inconsistently using `:` and also whether to capitalize the first word of the description. Pretty sure that's throughout the codebase though so nothing new. :)

**Review by jlebon — APPROVED** [review]
Thanks!

File: vars/gangplank.groovy (hunk 2 above)
[the remark this answers is not in the document]
Comment (ravanelli) [reply vars/gangplank.groovy:12]: @darkmuggle I took the [suggestion](https://github.com/coreos/coreos-ci-lib/pull/71#discussion_r628292282) from @jlebon after getting an issue with the `finalize` function, it indeed had a naming collision.

**Review by jlebon — APPROVED** [review]

**Pull request merged**